dapps/bcc/dbcp.json

```diff
@@ -13,7 +13,7 @@
       "type": "library"
     },
     "dbcpVersion": 2,
-    "description": "Contractus for loading ens entries and it's data...",
+    "description": "Contractus for loading ens entries and it's data",
```

Comment:
is by `Contractus` the old company name of evan meant or `Contracts` ?

Reply:
```suggestion
    "description": "Contracts for loading ens entries and their data",
```

Reply:
Almost @karladler :p